src/config.rs

```diff
@@ -287,12 +287,11 @@ impl Config {
             }
         });
 
-        let level = match config.verbose {
-            0 => log::LevelFilter::Error,
-            1 => log::LevelFilter::Warn,
-            2 => log::LevelFilter::Info,
-            _ => log::LevelFilter::Debug,
-        };
+        if config.verbose > 0 {
+            eprintln!("Error: please use `log_filters` to set logging verbosity",);
+            std::process::exit(1);
```

Comment:
Shouldn't release notes say "removed" instead of "deprecated"?

Reply:
Updated the PR and the relevant commits.